controllers/metal3machine_controller.go

```diff
@@ -207,6 +230,7 @@ func (r *Metal3MachineReconciler) reconcileNormal(ctx context.Context,
 	// Make sure bootstrap data is available and populated. If not, return, we
 	// will get an event from the machine update when the flag is set to true.
 	if !machineMgr.IsBootstrapReady() {
+		machineMgr.SetConditionMetal3MachineToFalse(capm3.AssociateBMHCondition, capm3.WaitingForBootstrapReadyReason, capi.ConditionSeverityInfo, "")
```

Comment:
same re: error missing comment.

Reply:
Same comment as above. Don't think this should be considered an error condition.